src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.kt

```diff
@@ -246,6 +236,11 @@ class DetectProperties {
             setHelp("A comma-separated list of values to be used with the Signature Scanner --exclude flag.", "Each pattern provided is passed to the signature scanner (Black Duck scan CLI) as a value for an --exclude option. The signature scanner requires that these exclusion patterns start and end with a forward slash (/) and may not contain double asterisks (**). These patterns will be added to the paths created from detect.blackduck.signature.scanner.exclusion.name.patterns and passed as --exclude values. Use this property to pass patterns directly to the signature scanner as-is. For example: suppose you are running in bash on Linux, and have a subdirectory named blackduck-common that you want to exclude from signature scanning. Any of the following would exclude it: --detect.blackduck.signature.scanner.exclusion.patterns=/blackduck-common/, --detect.blackduck.signature.scanner.exclusion.patterns='/blackduck-common/', --detect.blackduck.signature.scanner.exclusion.patterns='/blackduck-*/'. Use detect.blackduck.signature.scanner.exclusion.name.patterns when you want Detect to convert the given patterns to actual paths.")
             setGroups(DetectGroup.SignatureScanner, DetectGroup.SourceScan)
         }
+        val DETECT_BLACKDUCK_SIGNATURE_SCANNER_INDIVIDUAL_FILE_MATCHING = EnumProperty("detect.blackduck.signature.scanner.individual.file.matching", IndividualFileMatchMode.NONE, IndividualFileMatchMode::class.java).apply {
+            setInfo("Individual File Matching", "6.2.0")
+            setHelp("Users may set this property to SOURCE, BINARY, ALL, or NONE, to indicate what types of files they want to match")
+            setGroups(DetectGroup.SignatureScanner)
+        }
```

Comment:
You should be using a FilterableEnumListProperty. Use the detect.included.detector.types property as an example.

Reply:
Also the help text should not include hard coded example values anymore. The new configuration system is aware of the potential values.